modules/change_password.php

```diff
@@ -0,0 +1,75 @@
+<?php
+$userId = isset($_SESSION['login_user_id']) ? $_SESSION['login_user_id'] : null;
+if (!$userId) {
+    header('Location:index.php');
+    exit();
+}
+
+$isSuccess = isset($_GET['success']) ? $_GET['success']: null;
+
+if (!empty($_POST)) {
+    $current = md5($_POST['current']);
+    $update = md5($_POST['update']);
+    $confirm = md5($_POST['confirming']);
+
+    $search_current_password = "SELECT password FROM users WHERE id = $userId LIMIT 1";
+    $result = $mysql->query($search_current_password);
+    $user = $result->fetch_array() ?? false;
+
+    $update_password = "UPDATE users SET password = '$update' WHERE id = $userId";
+    $errors = [];
+    
```

Comment:
You need to handle some error cases and push error message in input:

1. Entered current Password not match with current password in DB
2. Entered confirm password not match with entered new password